components/form-builder/app/DownloadTable.tsx

```diff
@@ -0,0 +1,259 @@
+import React from "react";
+import { VaultSubmissionList } from "@lib/types";
+import { useTranslation } from "react-i18next";
+import { ExclamationText } from "./shared";
+import { SkipLinkReusable } from "@components/globals/SkipLinkReusable";
+
+interface DownloadTableProps {
+  submissions: VaultSubmissionList[];
+  checkedItems: Map<string, boolean>;
+  setCheckedItems: React.Dispatch<React.SetStateAction<Map<string, boolean>>>;
+  getCheckedItems: () => Map<string, boolean>;
+  handleDownload: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+export const DownloadTable = ({
+  submissions,
+  checkedItems,
+  setCheckedItems,
+  getCheckedItems,
+  handleDownload,
+}: DownloadTableProps) => {
+  const { t } = useTranslation("form-builder");
+  const DOWNLOAD_OVERDUE = 15;
+  const CONFIRM_OVERDUE = 15;
```

Comment:
Values based on this table. My interpretation should probably be double checked :)
https://docs.google.com/document/d/1CxfOUivb4tBtD-7Y2BZydM-r3rITgaz-q9OwbNz2_Fk/edit

Reply:
There was specific Github issues created for this work because it was part of the Nagware design documentation. I tried to update it as much as possible with the latest update on the document you linked above.

For reference:
https://github.com/cds-snc/platform-forms-client/issues/1026
https://github.com/cds-snc/platform-forms-client/issues/1027
https://github.com/cds-snc/platform-forms-client/issues/1028

You may also want to close corresponding tasks when merging the pull request.